README.md

```diff
@@ -98,13 +98,28 @@ Simply:
 2. Run: `gollum`.
 3. Open `http://localhost:4567` in your browser.
 
-If you wish to run from source, simply:
+This will start up a web server (WEBrick) running Gollum with a web interface, where you can view and edit your wiki.
 
-1. Navigate to your git repository (wiki) via the command line.
-2. Run: `bundle exec bin/gollum`
-3. Open `http://localhost:4567` in your browser.
+### Running from source
 
-This will start up a web server (WEBrick) running Gollum with a web interface, where you can view and edit your wiki.
+Running from the project folder (the easy and comfortable way):
+
+1. `git clone https://github.com/gollum/gollum`
+2. `cd gollum`
+3. `[sudo] bundle install` (may not always be necessary).
+4. `bundle exec bin/gollum`
+5. Open `http://localhost:4567` in your browser.
+
+Running from outside the project folder (may be necessary sometimes?):
```

Comment:
I think these instructions for running outside of the project folder are not necessary for the readme. It's not really gollum specific, but rather some shell handiness that people should be able to figure out themselves.